projects/cdk/utils/dom/get-selection-text.ts

```diff
@@ -0,0 +1,23 @@
+type Target = Element | EventTarget | null;
+
+/**
+ * @description:
+ * cross browser way to get selected text
+ *
+ * History:
+ * BUG - window.getSelection() fails when text selected in a form field
+ * https://bugzilla.mozilla.org/show_bug.cgi?id=85686
+ */
+export function tuiGetSelectionText(windowRef: Window, target?: Target): string | null {
```

Comment:
What about copying not from an input field? There's got to be target in that event as well.

Reply:
fixed